gst_plugins/python/savant_rs_video_demux.py

```diff
@@ -408,6 +429,25 @@ def handle_eos(self, eos: EndOfStream) -> Gst.FlowReturn:
 
         return Gst.FlowReturn.OK
 
+    def handle_shutdown(self, shutdown: Shutdown) -> Gst.FlowReturn:
+        """Handle Shutdown message."""
+        if self.shutdown_auth is None or shutdown.auth != self.shutdown_auth:
+            self.logger.debug('Ignoring shutdown message.')
+            return Gst.FlowReturn.OK
+
+        self.logger.info('Received shutdown message.')
```

Comment:
Let's make logging a bit more detailed here by adding the reason the shutdown message was ignored.